javascript/javascript1/week3/Js1week3.js

```diff
@@ -0,0 +1,189 @@
+
+
+//Series duration of my life
+
+const seriesDurations = [
+    {
+        title: "friends",
+        days: 4,
+        hours: 2,
+        minutes: 20,
+    },
+    {
+        title: "Breaking Bad",
+        days: 6,
+        hours: 0,
+        minutes: 50,
+    },
+    {
+        title: "Modern family",
+        days: 10,
+        hours: 9,
+        minutes: 0,
+    },
+];
+expectedAge = 80 * 365;
+function percentageOfAge(array) {
+    let sum = 0;
+    for (i = 0; i < array.length; i++) {
+        const totalDayFormat = [];
+        const percentage = [];
+        totalDayFormat[i] = (array[i].days + (array[i].hours / (24)) + (array[i].minutes / (24 * 60)));
+        percentage[i] = (totalDayFormat[i] / expectedAge) * 100;
+        result = console.log(`${array[i].title} took ${percentage[i].toFixed(3)}% of my life.`);
+        sum += totalDayFormat[i];
+        totalPercentage = (sum / expectedAge) * 100;
+
+    }
+    console.log(`In total that is ${totalPercentage.toFixed(3)} of my life`);
+    return result;
+}
+percentageOfAge(seriesDurations)
+
+const notes = [];
+function saveNote(content, id) {
+    return notes.push({ content, id });
+
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("water the plants", 3);
+
+
+
+function getNote(id) {
+    if (typeof id !== 'number' || typeof id === null || id > notes.length) {
+        return console.log(`It is an error string`);
+    }
+
+    else {
+
+        for (let i = 0; i <= notes.length; i++) {
+
+            if (id == notes[i].id) {
+                return console.log(notes[i])
+            }
+
+        }
+
+    }
+}
+
+getNote(3)
+//
+function logOutNotesFormatted() {
+    for (let i = 0; i < notes.length; i++) {
+
+        noteFormat = console.log(`The note with id: ${notes[i].id}, has the following note text: ${notes[i].content}.`)
+    }
+    return noteFormat;
+}
+logOutNotesFormatted()
+
+//  Usage limit
+let activities = [];
+function addActivity(date, activity, duration) {
+    if (typeof date === 'string' && typeof activity === 'string' && typeof duration === 'number')
+
+        return activities.push({ date, activity, duration })
+    else
+        return console.log(`It is not a valid data type.`)
+}
+
+
+addActivity("07/6-22", "Youtube", 30);
+addActivity("07/6-22", "TikTok", 30);
+addActivity()
+
+//addActivity("07/06-22", "Twitter", 60);
+
+function showStatus(activities) {
+    let activitySum = 0;
+    for (let i = 0; i < activities.length; i++) {
+        activitySum += activities[i].duration;
+    }
+    if (activities.length == 0) {
+        return console.log(`Add some activities before calling showStatus.`);
+    }
+
+    else if (activitySum <= 120) {
+
+        return console.log(`You have added ${activities.length} activities. They amount to ${activitySum} min. of usage`)
+    }
+    else
+        return console.log(`You have reached your limit, no more smartphoning for you!`)
+
+}
```

Comment:
You should not return your console.log statements. Simply omit the return keyword from line 144, 149 and 152. 

About return statements in Javascript 
> https://www.w3schools.com/jsref/jsref_return.asp

About console.log() in Javascript 
> https://developer.mozilla.org/en-US/docs/Web/API/console/log